### DISCUSSION TITLE
Is there any possible way to debug ALVR server in VS 2019?

### DISCUSSION THREAD

**Discussion by xerosis** [alvr-org/ALVR#453] [Q&A] [answered]
Hello, I've been struggling to debug alvr server. ex) FrameRender.cpp
Is there any way to debug alvr server app w/ setting breakpoints in VS 2019?
Any help would be appreciated. 

Huge thanks for maintaining this lovely VR streaming solution!

**Comment by zmerp** [comment, marked as answer]
No sorry, that is not possible. There is a Visual Studio solution in alvr/server/cpp but that has become non functional from ALVR v12. ALVR uses Rust. You can use VS Code and the LLVM extension for debugging but that only works with the Rust code, not C++.

**Reply by xerosis** [reply]
That makes much sense, thank you :)
